modules/gatsby/package.json

```diff
@@ -62,6 +62,7 @@
     "gatsby-transformer-sharp": "2.1.8",
     "inquirer": "^6.2.1",
     "lodash": "^4.17.4",
+    "lodash.debounce": "^4.0.8",
```

Comment:
For different PR: Check if we can drop lodash and gatsby-plugin-lodash in favor or just the specific plugins. The starter I based this on used lodash but I don't think we use lodash anymore,